### PR TITLE
Users/riyanshgoyal/aad network access forwarding profile

### PR DESCRIPTION
**Pull request by RiyanshGoyal**
#### Pull Request (PR) description
Initial Release
 - AADNetworkAccessForwardingProfile

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

**Comment by RiyanshGoyal** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by RiyanshGoyal** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: Modules/Microsoft365DSC/Dependencies/Manifest.psd1 [hunk 1]
```diff
@@ -56,6 +56,10 @@
             ModuleName      = 'Microsoft.Graph.Beta.DeviceManagement.Enrollment'
             RequiredVersion = '2.24.0'
         },
+        @{
+            ModuleName      = 'Microsoft.Graph.Beta.NetworkAccess'
+            RequiredVersion = '2.23.0'
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Dependencies/Manifest.psd1]: Please update this to be version 2.24.0.

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
One small comment

File: Modules/Microsoft365DSC/Dependencies/Manifest.psd1 (hunk 1 above)
Comment (RiyanshGoyal) [reply Modules/Microsoft365DSC/Dependencies/Manifest.psd1]: updated. Thanks

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**